### PR TITLE
Add missing targets

### PR DESCRIPTION
**Pull request by Wolf480pl**
Add the following targets which were definitely missing:
* hakasie
* tap (tapping asie's head)

DISCLAIMER: untested

### CODE REVIEW

**Comment by Michcioperz** [comment]
Got nothing against it. Can I have @asiekierka's seal of approval here?

**Comment by Michcioperz** [comment]
Well whatever this can't wait ¯_(ツ)_/¯

File: nnnnnnaas.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
            "txt": lambda ret: Response("\n\n".join(["".join(x) for x in ret]), mimetype="text/plain"),
            "html": lambda ret: render_template("gimme.html", ret=ret, choice=choice)}
 
-TEXTS = {"nano": ["NANO"], "hakase": ["HAKASE"], "adore": ["NANO", "HAKASE"], "asie": ["ASIE"]}
+TEXTS = {"nano": ["NANO"], "hakase": ["HAKASE"], "adore": ["NANO", "HAKASE"], "asie": ["ASIE"], "hakasie": ["HAKASIE"],
+         "tap": ["*taps asie's head*", "dlaczego mogę ją tapować po łebku?"]}
```
Comment (asiekierka) [inline nnnnnnaas.py:18]: Does NNNNNNaaS handle UTF-8 properly?

Comment (Michcioperz) [reply nnnnnnaas.py:18]: I suppose someone might want to make a pull request
Nah I bet it does

**Pull request merged**